chainer/functions/connection/convolution_2d.py

```diff
@@ -388,6 +398,8 @@ def convolution_2d(x, W, b=None, stride=1, pad=0, cover_all=False, **kwargs):
             ``pad=p`` and ``pad=(p, p)`` are equivalent.
         cover_all (bool): If ``True``, all spatial locations are convoluted
             into some output pixels.
+        dilate (int or pair of ints): Dilation factor of filter applications.
+            ``dilate=d`` and ``dilate=(d, d)`` are equivalent.
```

Comment:
How about putting a reference to `DilatedConvolution2DFunction` class? I guess dilated convolution is not familiar with beginners.